packages/manager/src/managers/SliceMachineManager.ts

```diff
@@ -172,6 +175,20 @@ export class SliceMachineManager {
 		return API_ENDPOINTS;
 	}
 
+	getMode(): (typeof APPLICATION_MODE)[keyof typeof APPLICATION_MODE] {
+		const mode = process.env.SM_ENV || APPLICATION_MODE.Production;
```

Comment:
I removed `getMode()` and replaced it with something that works more like the existing API_ENDPOINTS constant:

https://github.com/prismicio/slice-machine/blob/4eba1b48beefce863269543783340d539d29d119/packages/manager/src/constants/GIT_PROVIDER_APP_SLUGS.ts

I agree that `API_ENDPOINTS` and the new `GIT_PROVIDER_APP_SLUGS` could be a function rather than a constant. Let's save that for a different PR (unless you feel strongly about it).